### PR TITLE
Add compatibility with PHPStan v1.0

### PR DESCRIPTION
**Pull request by porozhnyy**
#47 
@marmichalski @akondas  Hey! Maybe you can check this PR?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,7 +15,8 @@
 ## Compatibility
 
 | PHPStan version | Yii2 extension version |
-| --------------- | ---------------------- |
+|-----------------|------------------------|
+| 1.0             | 1.x                    |
```
Comment (marmichalski) [inline README.md:19]: I would rather not bump this extension version to 1.x 🤔

**Review by marmichalski — COMMENTED** [review]
CI failed.
I am ok with ignoring the errors related to backward compatibility promise, that's what I did as I have also been working on bringing phpstan 1.x support, but due to time constraints did not finish yet 😿.

File: README.md (hunk 1 above)
Comment (porozhnyy) [reply README.md]: Done

**Comment by porozhnyy** [comment]
> CI failed. I am ok with ignoring the errors related to backward compatibility promise, that's what I did as I have also been working on bringing phpstan 1.x support, but due to time constraints did not finish yet crying_cat_face.

Due to lack of time, I also do not have time to fix them, I decided to open PR in the hope that someone will correct them. I will try to find the time and solve all the problems in the coming days

**Comment by marmichalski** [comment]
Done in #50.

**Pull request closed without merging**